src/var/Var.jl

```diff
@@ -12,17 +12,28 @@ using Bio.Seq
 
 export
 
-    ## Mutation Counting
-
-    ### Mutation count types.
+    # Mutation counting
+    MutationType,
     DifferentMutation,
     TransitionMutation,
     TransversionMutation,
 
-    ### Checking mutation types of nucleotides.
+    count_mutations,
+
+    # Genetic and Evolutionary distances
+    Count,
+    Proportion,
+    JukesCantor69,
+    Kimura80,
+    JC69,       # alias for JukesCantor69
+    K80,        # alias for Kimura80
```

Comment:
I guess these type aliases are already removed (a peculiar fact is that Julia doesn't warn you for this kind of mistakes).